Uniswap/interface
# feat: use QuoterV2

# Description

- Addresses an [open ticket](https://uniswaplabs.atlassian.net/browse/INFRA-88?atlOrigin=eyJpIjoiZGFjZDZkYmRkYTI2NGM2Mzg2NzAzNzU0Y2UyMjkyOWMiLCJwIjoiaiJ9) where we only use QuoterV2 for Celo
- QuoterV2 address can be found here: https://docs.uniswap.org/contracts/v3/reference/deployments
- Removes the option to use a `Quoter` type on the `useQuoter` hook
- Renames the Quoter addresses to v2 as required
- Includes unit tests for the client-side trade hook to make sure the Quoter version stays the same
- FYI - AutoRouter already uses QuoterV2

# Manual test
- Navigate to the interface (preview link on the PR), on the swap page turn **OFF** the auto router option and make sure the quotes are accurate.
